competitions/template/SubversionREADME.tmpl

```diff
@@ -0,0 +1,36 @@
+Please only store encrypted data in this directory.                   
```

Comment:
This is the template for the README that goes in the competition's bigdata repository directory, right?

I think just calling it "SubversionREADME.tmpl" might be misleading, or at least ambiguous.

Reply:
How about `DATA_README.tmpl`?